src/app/app.routing.ts

```diff
@@ -148,7 +151,27 @@ export const routes: Routes = [
         path: 'unlocked-stage/game',
         component: UnlockedStageComponent,
         canActivate: [LoggedInGuard]
-    }
+    },
+    {
+        path: 'prevent',
+        canActivate: [LoggedInGuard],
+        component: PreventionComponent,
+        children: [
+            {
+              path: '',
+              redirectTo: 'stop-the-breed/game',
+              pathMatch: 'full'
+            },
+            {
+              path: 'stop-the-breed/game',
+              component: StopTheBreedComponent,
+            },
+            {
+            path: 'stride-against-malaria/game',
```

Comment:
spacing.

Reply:
They are routes to children components, that's why relatively more spacing.